qa/tasks/keycloak.py

```diff
@@ -0,0 +1,327 @@
+"""
+Deploy and configure Keycloak for Teuthology
```

Comment:
@TRYTOBE8TME Which file in `qa/suites/rgw/sts/tasks` is calling this keycloak task to run this code? I can't find it in this PR.

Reply:
The webidentity.yaml file in qa/suites/rgw/sts/tasks calls keycloak task.